### DISCUSSION TITLE
add hail / Severe Thunderstorms

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17721] [Feature requests]
Currently we have thunderstorms, but I think we should add Severe Thunderstorms with Gusty Wind noises and Hail (similar to snow but falling down faster and not wavy) and have more frequent lightning and thunder. For gameplay this could make rides break down, similar to how parkitect has "struck by lightning" on multiple rides when it has storms

**Comment by TheClaudeQC** [comment]
it could be an interesting idea, especially whit hot and humid.

**Comment by astraylife1** [comment]
Or have all the rides shut down in severe weather

**Comment by Derpercoaster** [comment]
This feels a bit too "different". You can only change this game so much before it feels like a different game, and loses the charm it has.

**Comment by 733737** [comment]
they already added blizzard with thunder, and snow to the game

**Reply by HtotheTML** [reply]
oh, they did?

**Reply by Derpercoaster** [reply]
> oh, they did?

Yep. in scenarios with a cold climate, rain , heavy rain, and thunderstorms are replaced with snow, heavy snow, and blizzards, respectively. The rain sound doesn't play (since snow is quiet I guess), and the falling rain looks more like snow. The weather icon in the bottom right also depicts a snowflake instead of raindrops.
